SrfGen/srfinfo2vm.py

```diff
@@ -792,7 +795,7 @@ def create_vm(args, srf_meta, logger_name: str = "srfinfo2vm"):
     )
 
     # proportion in ocean
-    if args.no_optimise or (xlen0 > 0 and ylen0 > 0):
+    if args.no_optimise or (xlen0 <= 0 and ylen0 <= 0):
```

Comment:
`x<=0 or y<=0`, otherwise if only one is negative it won't optimise? (De Morgans law?)

Reply:
It was meant to be a specific case handling a few lines up. But actually if ever either is 0 then the area is 0 then we can't really optimise it.

Reply:
Also needs to be `not (x<=0 or y<=0)` for the logic to work out. Slowly remembering De Morgans Law (negate outside, flip conjunctive, negate particles)

Reply:
The logic now is slightly different but the cases that it doesn't optimise for are now more correct.